File: app/src/main/java/example/com/crackle/model/CertificationResults.java
package example.com.crackle.model;

import com.google.gson.annotations.SerializedName;

import java.util.List;

public class CertificationResults {

    @SerializedName("countries")
    private
    List<Certification> certificationList;

    public CertificationResults(List<Certification> certificationList) {
        this.certificationList = certificationList;
    }

    public List<Certification> getCertificationList() {
        return certificationList;
    }
}

File: app/src/main/java/example/com/crackle/model/CertificationResults.kt
package example.com.crackle.model

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.android.parcel.Parcelize

@Parcelize
data class CertificationResults(@field:SerializedName("countries")
                           val certificationList: List<Certification>) : Parcelable
